ui/admin/tests/e2e/README.md

```diff
@@ -0,0 +1,44 @@
+# Admin UI e2e tests
+
+This test suite tests the Boundary Admin UI in an end-to-end setting. It was designed to be run in a
+variety of environments as long as the appropriate environment variables are set. You can choose to
+spin up your own infrastructure or use [Enos](https://github.com/hashicorp/boundary/tree/main/enos)
+to generate it for you. There is an enos scenario in the `boundary` repository that runs these
```

Comment:
Can we link to the scenario on the text "an enos scenario"?
```suggestion
to generate it for you. There is [an enos scenario](/) in the `boundary` repository that runs these
```

Reply:
It hasn't been created yet. I'll remove this line and update it once it's there.